termios/pty_bsd.go

```diff
@@ -0,0 +1,26 @@
+package termios
```

Comment:
You need to add
```
// +build darwin freebsd openbsd netbsd

```
Above the `package` declaration to scope this file to bsd like things (naming the file _bsd.go doesn't do anything by itself)